data_fixes/C1491_restrictions/remove_restrictions.rb

```diff
@@ -0,0 +1,27 @@
+require 'archivesspace/client'
+require 'json'
+require 'csv'
+require_relative '../../helper_methods.rb'
+
+aspace_login()
+
+start_time = "Process started: #{Time.now}"
+puts start_time
+
+csv = CSV.parse(File.read("C1491_restrictions_curis.csv"), :headers => true)
+log = "log_delete_restrictions_C1491.txt"
+
+csv.each do |row|
+  uri = row['uri']
+  ao = @client.get(uri).parsed
+
+notes = ao.dig('notes')
+notes.delete_if { |note| note["type"] == "accessrestrict" }
+post = @client.post(uri, ao.to_json)
+response = post.body
+puts response
+File.write(log, response, mode: 'a')
+
+rescue Exception => msg
+puts "Processing failed at #{Time.now} with error '#{msg.class}: #{msg.message}'"
```

Comment:
Would it make sense to log this error message, as well as outputting it to the terminal?

Reply:
Hmmmm. I mostly use the log in this type of process to have a record of what was changed, in case the stakeholders have questions after the fact. I wouldn't want it in the same log file, so perhaps a separate one. This process has already run, so perhaps I'll give that some thought next time if you're ok with that?